Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.ts

```diff
@@ -90,40 +104,56 @@ class BocAutoCompleteReferenceValue //TODO RM-7715 - Make the TypeScript classes
       // The remaining browsers support ARIA 1.1 with NVDA.
 
       var internetExplorerScreenReaderLabelID = baseID + '_InternetExplorerScreenReaderLabel';
-      var screenReaderLabel = $("<span hidden='hidden' />")
-        .attr("id", internetExplorerScreenReaderLabelID)
-        .html(resources.InternetExplorerScreenReaderLabelText)
-        .appendTo(combobox);
-
-      this._textbox.attr('role', combobox.attr('role'));
-      combobox.removeAttr('role');
-
-      this._textbox.attr('aria-haspopup', combobox.attr('aria-haspopup'));
-      combobox.removeAttr('aria-haspopup');
+      var screenReaderLabel = document.createElement("span");
+      screenReaderLabel.hidden = true;
+      screenReaderLabel.id = internetExplorerScreenReaderLabelID;
+      screenReaderLabel.innerText = resources.InternetExplorerScreenReaderLabelText;
```

Comment:
```suggestion
      screenReaderLabel.innerHTML = resources.InternetExplorerScreenReaderLabelText;
```

There's both APIs in jquery and in DOM. We intentionally use HTML not Text here.